### PR TITLE
Updated PT-BR lang file

### PR DESCRIPTION
**Pull request by AFaustini**

### CODE REVIEW

File: lang/1046.json [hunk 1]
```diff
@@ -199,5 +200,28 @@
         "show_picotorrent_in_notification_area": "Mostrar o PicoTorrent na area de notificação",
         "minimize_to_notification_area": "Minimizar para area de notificação",
         "close_to_notification_area": "Fechar para area de notificação"
```
Comment (vktr) [inline lang/1046.json]: This line needs to end with a comma `,`

**Comment by vktr** [comment]
Awesome! 😄

**Comment by AFaustini** [comment]
Before the final release, please call me to update the translation.

**Comment by vktr** [comment]
Will do! I really appreciate your help!

**Pull request merged**